util/app-config/src/exit_code.rs

```diff
@@ -48,3 +49,10 @@ impl From<clap::Error> for ExitCode {
         ExitCode::Cli
     }
 }
+
+impl From<ParseBoolError> for ExitCode {
+    fn from(err: ParseBoolError) -> ExitCode {
+        eprintln!("Config Error: {:?}", err);
```

Comment:
A story before getting to the point:
I had a bad experience with [`capsule`].
When I ran [`capsule`] in my laptop, it always threw "file not found", but I never had a chance to know which file was not found.
Even I enabled `RUST_BACKTRACE=full`, I still didn't know where the error was thrown.
Ref: [How `capsule` handle the errors?](https://github.com/nervosnetwork/capsule/blob/56bf934d89824c03a0464f347d2f24f3558d9d6d/src/bin/capsule.rs#L406-L409)

So, if I was a ckb user, I would rather know the incorrect field than only know why has an error.
My suggestion here is: remove this `From` , construct the error at the location where throws the error and write the filed name into the error message.

[`capsule`]: https://github.com/nervosnetwork/capsule

Reply:
good point!